Assets/Scripts/GUI/BasePanel.cs

```diff
@@ -129,6 +130,19 @@ private enum FixedTransitionState
 
         [SerializeField] protected PopupMapKey[] m_PanelPopUpMap;
         [SerializeField] protected string m_PanelDescription;
+        [SerializeField] protected LocalizedString m_LocalizedPanelDescription;
+
+        public string PanelDescription
+        {
+            get
+            {
+                if (m_LocalizedPanelDescription.TableReference != null)
+                {
+                    return m_LocalizedPanelDescription.GetLocalizedString();
+                }
+                return m_PanelDescription;
```

Comment:
This was partially to catch myself for missed panels to try to prevent null reference exceptions, but it seems to perform better than expected! Ideally we no longer need m_PanelDescription and similar things across the codebase.